packages/core/src/scripts/migrate-oz-cli-project.ts

```diff
@@ -0,0 +1,426 @@
+
+  console.log("\nThese were your project's compiler options:");
+  console.log(compiler);
+}
+
+async function migrateManifestFiles() {
+  const oldManifests = await getManifests();
+  const migratableManifests = oldManifests.filter(manifest => !isDevelopmentNetwork(getNetworkName(manifest)));
+  const { newManifests, manifestsMigrationData } = await migrateManifests(migratableManifests);
+
+  for (const manifestFile in newManifests) {
+    const newManifest = newManifests[manifestFile];
+    const newFileName = getNewManifestLocation(manifestFile);
+    await writeJSONFile(newFileName, newManifest);
+    console.log(SUCESS_CHECK + `Successfully migrated ${newFileName}`);
+  }
+
+  return manifestsMigrationData;
+}
+
+async function deleteLegacyFiles(): Promise<void> {
+  const oldManifests = await getManifests();
+  const developmentManifests = oldManifests.filter(manifestFile => isDevelopmentNetwork(getNetworkName(manifestFile)));
+
+  for (const manifestFile of developmentManifests) {
+    console.log(SUCESS_CHECK + `Deleting unused development manifest ${manifestFile}`);
+    await fs.unlink(manifestFile);
+  }
+
+  console.log(SUCESS_CHECK + `Deleting ${PROJECT_FILE}`);
+  await fs.unlink(PROJECT_FILE);
+}
+
+async function migrateManifests(oldManifests: string[]): Promise<MigrationOutput> {
+  const manifestsMigrationData: Record<string, unknown> = {};
+  const newManifests: Record<string, ManifestData> = {};
+
+  for (const manifestFile of oldManifests) {
+    const oldManifest = JSON.parse(await fs.readFile(manifestFile, 'utf8'));
+
+    if (oldManifest.manifestVersion === undefined) {
+      throw new Error('Migration failed: manifest version too old');
+    }
+
+    const [major, minor] = oldManifest.manifestVersion.split('.');
+
+    if (major === '3') {
+      continue;
+    }
+
+    // 2.2 latest version
+    if (major !== '2' || minor !== '2') {
+      throw new Error(`Migration failed: expected manifest version 2.2, got ${major}.${minor} instead`);
+    }
+
+    const network = getNetworkName(manifestFile);
+    newManifests[manifestFile] = updateManifest(oldManifest);
+    manifestsMigrationData[network] = getExportData(oldManifest);
+  }
+
+  return {
+    newManifests,
+    manifestsMigrationData,
+  };
+}
+
+async function getManifests(): Promise<string[]> {
+  const files = await fs.readdir(OPEN_ZEPPELIN_FOLDER);
+  return files.filter(isManifestFile).map(location => path.join(OPEN_ZEPPELIN_FOLDER, location));
+}
+
+async function getProjectFile(): Promise<LegacyProjectFile> {
+  return JSON.parse(await fs.readFile(PROJECT_FILE, 'utf8'));
+}
+
+function isManifestFile(fileName: string): boolean {
+  const network = getNetworkName(fileName);
+  return isPublicNetwork(network) || isDevelopmentNetwork(network) || isUnknownNetwork(network);
+}
+
+function getNetworkName(fileName: string): string {
+  return path.basename(fileName, '.json');
+}
+
+function isDevelopmentNetwork(network: string): boolean {
```

Comment:
1337 should be removed. I was mistaken before, I thought 1337 would show up with `geth --dev`, but its default network id is 1. Ganache would never have a network id of 1337 unless it was manually set (and we wouldn't support that case).